content/en/llm_observability/setup/api.md

```diff
@@ -144,7 +144,13 @@ If the request is successful, the API responds with a 202 network code and an em
 | value   | string | Input or output value. This should be used for all spans except for LLM spans. |
 | messages| [[Message](#message)] | List of messages. This should only be used for LLM spans. |
 
+**Note**: When only `input.messages` is set for an LLM span, Datadog infers `input.value` from `input.messages` and uses the following inference logic:
+
+1. If a message with `role=user` exists, the content of the last message is used as `input.value`. 
+1. If a `user` role message is not present, the content is inferred by concatenating all messages that do not have the `role=system` or `role=function`.
```

Comment:
Just recently updated the logic to be
- "input.value is inferred by concatenating the `content` field of all messages"
(so there's no role exclusion)